src/Game.java

```diff
@@ -1,45 +1,119 @@
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Map;
 import java.util.Scanner;
+import java.util.function.Function;
+import java.util.stream.Collectors;
 
 public class Game {
-    public static void main(String[] args) {
-        System.out.println("Welcome to PPTHangman!");
-        System.out.println("Please enter the number of players:");
 
-        Scanner scanner = new Scanner(System.in);
+  public static Map<String, Integer> leaderboardToSet(List<Player> allPlayers) {
```

Comment:
leaderboardToSet sounds like "a function that takes a leaderboard and turns it into a set". What this function does is turn a list of players, into a mapping of their scores. Should be called something like "getPlayerScoresMap" or shorter.